src/CMakeLists.txt

```diff
@@ -102,3 +103,11 @@ set_target_properties(hotspot
 
 install(TARGETS hotspot RUNTIME DESTINATION ${KDE_INSTALL_BINDIR})
 install(FILES hotspot.notifyrc DESTINATION ${KDE_INSTALL_KNOTIFY5RCDIR})
+
+add_executable(authhelper authhelper.cpp)
+target_link_libraries(authhelper KF5::AuthCore)
+
+install(TARGETS authhelper DESTINATION ${KAUTH_HELPER_INSTALL_DIR})
```

Comment:
is the directory namespaced by `hotspot`? otherwise this name is far too generic, it should be called `hotspot-perf-authhelper` or similar